arbor/morph/region.cpp

```diff
@@ -306,7 +307,376 @@ std::ostream& operator<<(std::ostream& o, const all_& t) {
     return o << "(all)";
 }
 
+// Region with all segments distal from another region
+
```

Comment:
The semantics of this operation are really unclear to me. Can it be expressed in terms of set operations, distances, and distality/proximity? Or do we need to know about the individual cables in the start region?